presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -151,6 +152,14 @@ void PrestoServer::run() {
             "Https Client Certificates are not configured correctly");
       }
       clientCertAndKeyPath = optionalClientCertPath.value();
+
+      auto internalSharedSecret =
+          systemConfig->internalCommunicationSharedSecret();
+      auto internalJwtEnabled = systemConfig->internalCommunicationJwtEnabled();
+      if (internalJwtEnabled && internalSharedSecret.empty()) {
+        VELOX_USER_FAIL(
+            "When internal JWT(internal-communication.jwt.enabled) authentication is enabled, a shared secret(internal-communication.shared-secret) is required.");
```

Comment:
Use an error message similar to the above one for Https client certificates so 

"Internal JWT Shared secrets are not configured correctly"

Reply:
I copied this from the Java error message. I will change it.